crypto/x509/x_pubkey.c

```diff
@@ -228,32 +232,57 @@ EVP_PKEY *X509_PUBKEY_get(const X509_PUBKEY *key)
 }
 
 /*
- * Now two pseudo ASN1 routines that take an EVP_PKEY structure and encode or
- * decode as X509_PUBKEY
+ * Now three pseudo ASN1 routines that take an EVP_PKEY structure and encode
+ * or decode as X509_PUBKEY
  */
 
-EVP_PKEY *d2i_PUBKEY(EVP_PKEY **a, const unsigned char **pp, long length)
+EVP_PKEY *d2i_PUBKEY_ex(EVP_PKEY **a, const unsigned char **pp, long length,
```

Comment:
Not for this PR.. but I will be changing the name from _ex to _with_libctx at some point soon..

Reply:
I assume you'll do that with `d2i_PrivateKey_ex` et al as well

Reply:
PEM_read_bio_PrivateKey_ex
PEM_read_PrivateKey_ex
PEM_read_bio_PUBKEY_ex
PEM_read_bio_Parameters_ex
RAND_priv_bytes_ex
RAND_bytes_ex
OCSP_RESPID_match_ex
OCSP_RESPID_set_by_key_ex
d2i_PrivateKey_ex
d2i_AutoPrivateKey_ex
d2i_PUBKEY_ex
evp_get_cipherbyname_ex
evp_get_digestbyname_ex